app/assets/stylesheets/buildout_design_system/components/table.scss

```diff
@@ -1,11 +1,11 @@
 .table {
-  thead th {
-    font-weight: 500;
-    --#{$prefix}table-bg: #{$storm-grey-100};
-    padding: 8px;
+  thead {
+    td {
+      font-weight: 500;
```

Comment:
The rows in the `thead` are going to be `th` to keep things semantically.

Also, inside of the `light.scss` you should be able to set the `$table-th-font-weight:` to the correct font-weight. I believe it should be `$font-weight-bold` or `$font-weight-semibold` to match the design system mocks